src/ui/windows/TogglDesktop/TogglDesktop/ui/Resources/DesignUpdate/ToolWindow.xaml

```diff
@@ -0,0 +1,30 @@
+<ResourceDictionary xmlns="http://schemas.microsoft.com/winfx/2006/xaml/presentation"
+                    xmlns:x="http://schemas.microsoft.com/winfx/2006/xaml"
+                    xmlns:mah="http://metro.mahapps.com/winfx/xaml/controls">
+    <ResourceDictionary.MergedDictionaries>
+        <ResourceDictionary Source="Typography.xaml" />
+    </ResourceDictionary.MergedDictionaries>
+    <Style TargetType="mah:MetroWindow" x:Key="Toggl.ToolWindow">
+        <Setter Property="ResizeMode" Value="NoResize" />
+        <Setter Property="TitleCharacterCasing" Value="Normal" />
+        <Setter Property="BorderThickness" Value="0" />
+        <Setter Property="GlowBrush" Value="Black" />

```

Comment:
I think Its better to be DynamicResource and changeable with theme, so dark them has light GlowBrush and vice versa

Reply:
I see and  in general both the GUI and XAML code look nice and clean.